netmiko/extreme/extreme_ssh.py

```diff
@@ -1,13 +1,123 @@
 """Extreme support."""
 from __future__ import unicode_literals
+import re
+import time
 from netmiko.cisco_base_connection import CiscoSSHConnection
 
 
+QUESTIONS = {
+    'save': 'Do you want to save configuration to',
+}
+
+
 class ExtremeSSH(CiscoSSHConnection):
     """Extreme support."""
+
     def session_preparation(self):
-        """Extreme requires enable mode to disable paging."""
+        """
+        Extreme requires enable mode to disable paging.
+        """
         self._test_channel_read()
-        self.enable()
         self.set_base_prompt()
         self.disable_paging(command="disable clipaging\n")
```

Comment:
I was wondering where that came from, they don't have any enable, nor configure mode. You drop right into the configuration with an 'admin' account (prefixed with #) or a user account without configuration privileges (prefixed with >). The enable here seems to be sending an empty default anyways, which just results in an extra empty write - I vote for taking it out.

This is with enable:
```
2017-03-27 22:26:36,230 - netmiko - DEBUG - read_channel: ExtremeXOS
Copyright (C) 1996-2016 Extreme Networks. All rights reserved.
This product is protected by one or more US patents listed at http://www.extremenetworks.com/patents along with their foreign counterparts.
==============================================================================

Press the <tab> or '?' key at any time for completions.
Remember to save your configuration changes.


exos-hostname.1 #
2017-03-27 22:26:36,333 - netmiko - DEBUG - read_channel:
2017-03-27 22:26:38,338 - netmiko - DEBUG - read_channel:
2017-03-27 22:26:38,338 - netmiko - DEBUG - read_channel:
2017-03-27 22:26:38,339 - netmiko - DEBUG - write_channel:

2017-03-27 22:26:38,441 - netmiko - DEBUG - read_channel:
exos-hostname.1 #
2017-03-27 22:26:38,546 - netmiko - DEBUG - read_channel:
2017-03-27 22:26:38,547 - netmiko - DEBUG - write_channel:

2017-03-27 22:26:38,548 - netmiko - DEBUG - _read_channel_expect read_data:
exos-hostname.1 #
2017-03-27 22:26:38,652 - netmiko - DEBUG - read_channel:
2017-03-27 22:26:38,653 - netmiko - DEBUG - write_channel: disable clipaging

2017-03-27 22:26:38,654 - netmiko - DEBUG - _read_channel_expect read_data: disable clipaging
exos-hostname.2 #
2017-03-27 22:26:38,956 - netmiko - DEBUG - read_channel:
```

This is the same excerpt without enable (note one less write before the clipaging command): 

```
2017-03-27 22:25:01,218 - netmiko - DEBUG - read_channel: ExtremeXOS
Copyright (C) 1996-2016 Extreme Networks. All rights reserved.
This product is protected by one or more US patents listed at http://www.extremenetworks.com/patents along with their foreign counterparts.
==============================================================================

Press the <tab> or '?' key at any time for completions.
Remember to save your configuration changes.


exos-hostname.1 #
2017-03-27 22:25:01,322 - netmiko - DEBUG - read_channel:
2017-03-27 22:25:03,327 - netmiko - DEBUG - read_channel:
2017-03-27 22:25:03,328 - netmiko - DEBUG - read_channel:
2017-03-27 22:25:03,328 - netmiko - DEBUG - write_channel:

2017-03-27 22:25:03,432 - netmiko - DEBUG - read_channel:
exos-hostname.1 #
2017-03-27 22:25:03,537 - netmiko - DEBUG - read_channel:
2017-03-27 22:25:03,642 - netmiko - DEBUG - read_channel:
2017-03-27 22:25:03,642 - netmiko - DEBUG - write_channel: disable clipaging

2017-03-27 22:25:03,644 - netmiko - DEBUG - _read_channel_expect read_data: disable clipaging
exos-hostname.2 #
2017-03-27 22:25:03,948 - netmiko - DEBUG - read_channel:
```